src/lib/vocab/space/space.events.ts

```diff
@@ -88,4 +88,33 @@ export const read_spaces: ServiceEventInfo = {
 	},
 };
 
-export const events: EventInfo[] = [create_space, read_space, read_spaces];
+export const delete_space: ServiceEventInfo = {
+	type: 'ServiceEvent',
+	name: 'delete_space',
+	params: {
+		$id: 'https://felt.social/vocab/delete_space_params.json',
+		type: 'object',
+		properties: {
+			space_id: {type: 'number'},
+			community_id: {type: 'number'},
+		},
+		required: ['space_id', 'community_id'],
+		additionalProperties: false,
+	},
+	response: {
+		$id: 'https://felt.social/vocab/delete_space_response.json',
+		type: 'object',
+		properties: {
+			space_id: {type: 'number'},
+		},
+		required: ['space_id'],
+		additionalProperties: false,
+	},
```

Comment:
```suggestion
	response: {
		$id: 'https://felt.social/vocab/delete_space_response.json',
		type: 'null',
	},
```

I think this is right, and then run `gro gen`.